daemon/algod/api/algod.oas3.yml

```diff
@@ -7192,5 +7192,6 @@
     {
       "name": "private"
     }
-  ]
+  ],
+  "x-original-swagger-version": "2.0"
```

Comment:
This file is generated by a webapp, I guess that app changed the output to include this line.

Reply:
ah `curl https://converter.swagger.io/api/convert` call, thanks for clarification.